.ci/azure/linux_ngraph_onnx.yml

```diff
@@ -97,13 +97,15 @@ jobs:
 
   - script: |
       set -e
-      apt-get update && apt-get install -y lsb-release && apt-get clean all
+      sudo apt-get update && sudo apt-get install -y lsb-release git git-lfs
```

Comment:
Need to install `git` and `git-lfs` as the model updating script uses them. It worked before as the model repository was already set up. If there was an update to the repo's sha or its folder, it would fail - as it did in this PR earlier.